app/views/Export/ExportTemplate.js

```diff
@@ -102,11 +102,21 @@ export const ExportTemplate = ({
               </View>
             )}
 
-            <Typography style={styles.exportSectionTitles}>
+            <Typography
+              style={[
+                styles.exportSectionTitles,
+                lightTheme && { color: Colors.primaryHeaderText },
+              ]}>
               {headline}
             </Typography>
             <View style={{ height: 8 }} />
-            <Typography style={styles.contentText}>{body}</Typography>
+            <Typography
+              style={[
+                styles.contentText,
+                lightTheme && { color: Colors.primaryText },
```

Comment:
These `lightTheme &&` are gross, but this component needs refactoring overall now to accommodate the new typography system, so this is an immediate fix.